components/concept/tabstray/build.gradle

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion rootProject.ext.build['compileSdkVersion']
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.build['minSdkVersion']
+        targetSdkVersion rootProject.ext.build['targetSdkVersion']
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation project(':browser-session')
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:${rootProject.ext.dependencies['kotlin']}"
+}
+
+archivesBaseName = "abstract-toolbar"
+
+apply from: '../../../publish.gradle'
+ext.configurePublish(
+        'org.mozilla.components',
+        'concept-tabstray',
```

Comment:
We have some inconsistency here now:

concept-toolbar -> abstract-toolbar
concept-engine -> engine
concept-tabstray -> concept-tabstray

Should we just stick to the latter and file a follow-up?

Reply:
Yeah. Let's file something and come up with a good naming scheme once we migrate from jcenter to our own repository. That's a good time to change the artifact names.

Regarding abstract-toolbar ... that's probably a relic. In the beginning I used the word "abstract" instead of "concept". But that didn't work well with Java package names that cannot contain the word "abstract". And yeah, engine exists before we had concept modules. :)

Reply:
#368